knip.config.ts

```diff
@@ -0,0 +1,23 @@
+import { KnipConfig } from "knip";
+
+const config: KnipConfig = {
+	ignore: ["./prettier.config.ts", "./stylelint.config.ts", "./dangerfile.ts"],
+	ignoreDependencies: [
+		"pg",
+		"stylelint-config-recess-order",
+		"stylelint-config-standard",
```

Comment:
```suggestion
	ignoreDependencies: [
		"pg",
```

I guess they should be handled by default 

https://github.com/what1s1ove/whatislove.dev/blob/main/knip.config.js

btw, add `pg` as ignoreDependencies for "apps/backend" workspace's key